src/views/shipments.js

```diff
@@ -218,15 +218,25 @@ export class shipments {
     let ordered = this.ordered[this.shipment.account.to._id][genericName(transaction.drug)]
     let qty = +transaction.qty[this.role.account]
     let exp = transaction.exp[this.role.account]
+    let destMess = ordered.destroyedMessage || ""
 
     if ( ! ordered || ! qty) return
 
+    let qtyToCheck = 10   
+    if(transaction.drug.brand) qtyToCheck = 1
+
     let minQty    = qty >= (+ordered.minQty || 10)
     let minExp    = exp ? new Date(exp) - Date.now() >= (ordered.minDays || 120)*24*60*60*1000 : !ordered.minDays
     let isChecked = this.isChecked[$index] || false //apparently false != undefined
 
-    if((minQty && minExp) == isChecked)
-      return console.log('minQty', minQty, qty, 'minExp', minExp, exp)
+    if((minQty && minExp) == isChecked){
+        if(destMess != ""){
+          this.snackbar.show(ordered.destroyedMessage)
+          return
+        } else {
+          return console.log('minQty', minQty, qty, 'minExp', minExp, exp)
+        }
+    }
```

Comment:
Two minor stylistic edits: 
- No need to use else {} if you have a return. 
- No need to set a default value to destroyed message if we aren't going to use it.

In this case lets remove
```
let destMess = ordered.destroyedMessage || ""
```

and change this to (i think i want to keep console.log until we have really tested this in production for longer):

```
if((minQty && minExp) == isChecked){
      if(ordered.destroyedMessage)
           this.snackbar.show(ordered.destroyedMessage)

      return console.log('minQty', minQty, qty, 'minExp', minExp, exp)
 }
```